### ISSUE TITLE
Banner Ads Issue

### ISSUE THREAD

**Issue by Nigrini** [pH7Software/pH7-Social-Dating-CMS#567]
I am trying to get an image displayed for the 336x280 holder.js

It works perfectly on the localhost test system. However for some reason it is not working on the published site.

![nnn](https://user-images.githubusercontent.com/43742116/58762397-05a3a780-8550-11e9-82bc-fe60d703ecfe.PNG)

It is possible to access the image by following the link, therefore I know it is not an access problem, but the image does not display in preview, and it does not display on the client side.

What is interesting though is that when I hover with my mouse over the preview text, my browser display a folder that does not exist - As can be seen in the next image. 

![vvvv](https://user-images.githubusercontent.com/43742116/58762643-2240df00-8552-11e9-9e9d-614dbf04d8db.PNG)

NB - I did try with **img data-src** as well as **img src**

**Comment by pH-7** [comment]
This looks to be a problem with the library. Can you check with the inspector tool with your Web browser to see if you have any errors in there?

**Comment by pH-7** [comment]
To work with holder.js library, the code should be
```js
<a href="#0"><img data-src="holder.js/728x90" alt="%site_name% by %software_name%" title="%site_name% powered by %software_name%" /></a>
```

**Comment by Nigrini** [comment]
I am sorry, I dont understand.

**This is the Holder.js tag that is provided by the script**
<a href="#0"><img data-src="holder.js/336x280" alt="%site_name% by %software_name%" title="%site_name% powered by %software_name%" /></a>

**Now if you want to display your own image in the space that was "reserved" by the Holder.js**
1. I took an image (store.png) with dimensions 336x280 and uploaded it to my ISP hosted site in folder ads
2. In the admin panel I change the tag that was provided by the ph7script to the following
<a href="#0"><img src="https://opensecret.co.za/ads/store.png/336x280" alt="%site_name% by %software_name%" title="%site_name% powered by %software_name%" /></a>

This results in nothing to be displayed. Not in the preview in admin panel, and nothing on the client side.

3. Change to <img src="https://opensecret.co.za/ads/store.png/336x280">

This resulted in (see image)
![rr](https://user-images.githubusercontent.com/43742116/58784231-31bc3880-85e3-11e9-860b-c25fe405c452.PNG)

So basically my question is what must the tag look like in order to display my image. And what can possibly be the reason why it display the image on the localhost, but not on the live hosted site.

**Comment by pH-7** [comment]
Okay, I see. You don't want to use holder.js anymore here but yours.

The image URL is wrong. Since you want to use your own image and not the JS placeholder, the URL cannot end by `336x280`. It would be not found otherwise (just check by yourself and you will see => https://opensecret.co.za/ads/store.png/336x280).

Use the one bellow instead:

```html
<a href="https://opensecret.co.za/signup"><img src="https://opensecret.co.za/ads/store.png" alt="My dating website!" title="Signup to Opensecret!" /></a>
```

**Comment by pH-7** [comment]
Further helps can be found here: https://www.w3schools.com/tags/tag_img.asp

**Comment by pH-7** [comment]
Further helps can be found here: https://www.w3schools.com/tags/tag_img.asp about the img tag. The sample code is a different one since it uses `holder.js`. Please ignore holder.js's specification when you use your own image.

**Comment by pH-7** [comment]
Oh, actually, it looks your server restricted the image to be displayed. 

I tried with your image hosted somewhere else and it works like a charm. 
```html
<a href="https://opensecret.co.za/signup">
    <img src="https://user-images.githubusercontent.com/1325411/58792332-43f2a280-85f4-11e9-975c-7bd057920df8.png" alt="My dating website!" title="Signup to Opensecret!" />
</a>
```

(you might have to ask your web host about this issue concerning the image).

**Comment by Nigrini** [comment]
Thank you very much - I was thinking that I am loosing it here with that image :)

**Comment by Nigrini** [comment]
Sorted - For some reason my SSL certificate did not appreciate me adding a folder and populating it with images.  Once I moved the files to the Images folder used by the script it was more than happy to display them.

Thanx for the help !
